src/rpc/net.cpp

```diff
@@ -840,7 +896,10 @@ static const CRPCCommand commands[] =
     { "network",            "clearbanned",            &clearbanned,            {} },
     { "network",            "setnetworkactive",       &setnetworkactive,       {"state"} },
     { "network",            "getnodeaddresses",       &getnodeaddresses,       {"count"} },
+
+    /* Not shown in help */
```

Comment:
nit: Would be good to either consistently mention this in every file where "hidden" is used or nowhere. Maybe just keep the newline?

Reply:
removed